pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -209,6 +245,10 @@ func fitsRequest(podRequest *preFilterState, nodeInfo *framework.NodeInfo, ignor
 		ignoredExtendedResources = sets.NewString()
 	}
 
+	if ignoredResourceGroups == nil {
+		ignoredResourceGroups = sets.NewString()
+	}
```

Comment:
can drop these... Len() and Has() can be run on nil string sets

Reply:
done